src/app/pages/index.js

```diff
@@ -24,19 +14,51 @@ const applyBasicPageHandlers = pipe(
   withPageWrapper,
   withContexts,
 );
+
 const applyErrorPageHandlers = pipe(withLoading, withPageWrapper, withContexts);
 
 export const ArticlePage = pipe(
   applyBasicPageHandlers,
   withVariant,
-)(_ArticlePage);
-export const ErrorPage = applyErrorPageHandlers(_ErrorPage);
-export const FrontPage = pipe(applyBasicPageHandlers, withVariant)(_FrontPage);
-export const MediaAssetPage = applyBasicPageHandlers(_MediaAssetPage);
-export const MostReadPage = applyBasicPageHandlers(_MostReadPage);
-export const PhotoGalleryPage = applyBasicPageHandlers(_PhotoGalleryPage);
-export const LiveRadioPage = applyBasicPageHandlers(_LiveRadioPage);
-export const OnDemandRadioPage = applyBasicPageHandlers(_OnDemandRadioPage);
-export const OnDemandTvPage = applyBasicPageHandlers(_OnDemandTvPage);
-export const StoryPage = applyBasicPageHandlers(_StoryPage);
-export const IdxPage = applyBasicPageHandlers(_IdxPage);
+)(loadable(() => import(/* webpackPrefetch: true */ './ArticlePage')));
+
+export const ErrorPage = applyErrorPageHandlers(
+  loadable(() => import('./ErrorPage')),
+);
+
+export const FrontPage = pipe(
+  applyBasicPageHandlers,
+  withVariant,
+)(loadable(() => import(/* webpackPrefetch: true */ './FrontPage')));
```

Comment:
Just checking `/* webpackPrefetch: true */` allows only certain, more popular page types to be loaded first? 

This is pretty cool.